packages/cli/src/commands/backend/buildImage.ts

```diff
@@ -20,15 +20,15 @@ import { paths } from '../../lib/paths';
 import { run } from '../../lib/run';
 
 export default async (imageTag: string) => {
-  const tempDistWorkspace = await createDistWorkspace(['example-backend'], {
+  const packageName: string = process.env.npm_package_name!;
+  const tempDistWorkspace = await createDistWorkspace([packageName], {
```

Comment:
Let's just grab the current package instead imo. Something like `(await fs.readJson(paths.resolveTarget('package.json'))).name` written nicely 😅

Reply:
Changed it 😃